packages/jest-core/src/__tests__/__snapshots__/watch.test.js.snap

```diff
@@ -50,6 +50,17 @@ Watch Usage
 ]
 `;
 
+exports[`Watch mode flows makes watch plugin initialization errors look nice 1`] = `
+"Failed to initialize watch plugin 'PATH_REPLACED':
```

Comment:
we could print a relative path (making it relative to `config.cwd` or `config.rootDir`)? this works as well, of course 🙂

Reply:
Will still need to replace the absolute path in the stack trace for the snapshot though. Want to do it regardless of that?

Reply:
that should also be relative. look at your screenshot from OP, it's relative

Reply:
Well, it's not relative in the test. Maybe it is if I use a regular fixture instead of the `doMock` that all the other watch tests used as well.

Reply:
Yup, that seems to work